packages/callbacks/tizen/project_def.prop

```diff
@@ -0,0 +1,40 @@
+# See https://docs.tizen.org/application/tizen-studio/native-tools/project-conversion
+# for details.
+
+APPNAME = tizen_interop_callbacks_plugin
+type = staticLib
+profile = common-4.0
+
+# Source files
+USER_SRCS += src/*.cc
+
+# User defines
+USER_DEFS = PROFILE=$(profile)
+USER_UNDEFS =
+USER_CPP_DEFS = FLUTTER_PLUGIN_IMPL CPP_PROFILE=$(profile)
+USER_CPP_UNDEFS =
+
+# Compiler flags
+USER_CFLAGS_MISC =
+USER_CPPFLAGS_MISC =
+
+ifdef DART_SDK_INCLUDE
+  ifeq ("$(wildcard $(DART_SDK_INCLUDE)/dart_api_dl.h)","")
+    $(error DART_SDK_INCLUDE is set, but $$DART_SDK_INCLUDE/dart_api_dl.h does not exist)
+  endif
+else
+  PUBLIC=$(filter %/tizen-common/public, $(patsubst -I"%",%,$(USER_C_OPTS)))
+  DART_SDK_INCLUDE=$(join $(PUBLIC), /../../../../dart-sdk/include)
+  ifeq ("$(realpath $(DART_SDK_INCLUDE))", "")
+    $(error Could not find directory `$(DART_SDK_INCLUDE)` while looking for find dart_api_dl.h. Please set DART_SDK_INCLUDE environment variable)
+  endif
+  DART_SDK_INCLUDE:=$(realpath $(DART_SDK_INCLUDE))
+  ifeq ("$(wildcard $(DART_SDK_INCLUDE)/dart_api_dl.h)","")
+    $(error Failed to determine the location of dart_api_dl.h. Tried in `$(DART_SDK_INCLUDE)`. Please set DART_SDK_INCLUDE environment variable)
+  endif
```

Comment:
This looks nice, and is probably better than [copying the whole header files](https://github.com/swift-kim/ffi_callback/tree/master/tizen/src) or using an unusual relative include like this:

```cpp
#include <../../../../../dart-sdk/include/dart_api_dl.h>
```

Would it be fine if we use this as a template for plugins that use the relevant functionality (such as the video_player_videohole plugin which is currently under review)?

Also is this script portable? I was able to build the plugin on a Windows host without any problem, so I guess it is.

Reply:
Should be pretty reusable. It relies on `flutter-tizen build` passing include path for `tizen-common/public` - which is needed for any Flutter C++ application to build. As long as flutter-tizen header files layout do not change this will work. And then, it could be enhanced to handle new situation.
The DART_SDK_INCLUDE branch might not be needed. I wanted to have a way to override that mechanism in case it would be failing.